gui/modes/CommandPressureMode.qml

```diff
@@ -0,0 +1,308 @@
+                Layout.fillHeight: true
+                Layout.fillWidth: true
+                Layout.minimumWidth: 200
+
+                StepCounter {
+                    id: rrStepCounter
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    Layout.minimumWidth: 150
+                    Layout.minimumHeight: 100
+                    Layout.topMargin: 10
+                    Layout.rightMargin: 10
+                    Layout.leftMargin: 10
+
+                    title: qsTr("RR")
+                    value: guiState.rr
+                    onValueModified: guiState.rr = value
+                }
+
+                StepCounter {
+                    id: peepStepCounter
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    Layout.minimumWidth: 150
+                    Layout.minimumHeight: 100
+                    Layout.topMargin: 10
+                    Layout.rightMargin: 10
+                    Layout.leftMargin: 10
+
+                    title: qsTr("PEEP")
+                    value: guiState.peep
+                    onValueModified: guiState.peep = value
+                }
+
+                StepCounter {
+                    id: pipStepCounter
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    Layout.minimumWidth: 150
+                    Layout.minimumHeight: 100
+                    Layout.topMargin: 10
+                    Layout.rightMargin: 10
+                    Layout.leftMargin: 10
+
+                    title: qsTr("PIP")
+                    value: guiState.pip
+                    onValueModified: guiState.pip = value
+                }
+
+                StepCounter {
+                    id: ierStepCounter
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    Layout.minimumWidth: 150
+                    Layout.minimumHeight: 100
+                    Layout.topMargin: 10
+                    Layout.rightMargin: 10
+                    Layout.leftMargin: 10
+
+                    title: qsTr("I:E")
+                    stepSize: 0.1
+                    value: guiState.ier
+                    onValueModified: guiState.ier = value
+
+                    textFromValue: function (value, locale) {
+                        return Number(value).toFixed(1)
+                    }
+                }
+            }
+
+            GridLayout {
+                id: scopeGridLayout
+                columnSpacing: 0
+                rowSpacing: -20
+                Layout.minimumHeight: 400
+                Layout.minimumWidth: 500
+                Layout.fillHeight: true
+                Layout.fillWidth: true
+                flow: GridLayout.TopToBottom
+                rows: 3
+
+                ScopeView {
+                    id: pressureView
+                    name: "Pressure [cmH2O]"
+                    // TODO: Are these reasonable lower and upper bounds?
```

Comment:
Note - don't have to change in this PR, but we have authoritative answers to the correct ranges and increments for all of these in Valispace now. Maybe just edit the TODO to get correct ranges and increments from Valispace.

Reply:
I kept the same as original for now and just moved to a separate file as I mentioned before, so I thought it was better to not change anything. Eventually,  we will look through the implementation of this mode and we will revise everything once we get the right ranges.